test/visibility/empty/exceptEverything.chpl

```diff
@@ -0,0 +1,17 @@
+module M {
+  proc foo() {
+    writeln("In foo");
+  }
+
+  proc boo() {
+    writeln("In boo");
+  }
+}
+
+module M2 {
+  use M except *;
```

Comment:
I would add a comment to this use statement and the one in your other new test indicating what we're testing here.  Something along the lines of "verifying that this doesn't allow any of M's symbols to be accessed in an unqualified manner.  This syntax is intended to allow qualified access to symbols that would not have been available for qualified access otherwise"